doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -608,11 +608,7 @@ binding position. Here is an example:
 More generally, the parameter can be a pattern, as in the following
 variant:
 
-.. coqtop:: none
-
-   Reset Initial.
-
-.. coqtop:: in
+.. coqtop:: in rest
```

Comment:
Looks like I made a typo in my suggestion:
```suggestion
.. coqtop:: in reset
```

Reply:
Ah, no problem.